src/corelibs/U2View/src/ov_msa/MultilineScrollController.cpp

```diff
@@ -177,11 +177,59 @@ int MultilineScrollController::getViewHeight() {
     return childrenScrollArea->height();
 }
 
+bool MultilineScrollController::checkBoundary() {
+    int firstBase = -1;
+    int prevFistBase = -1;
+    bool needUpdate = false;
+    int childrenCount = ui->getChildrenCount();
+    int alignmentLen = maEditor->getAlignmentLen();
+    for (int i = 0; i < childrenCount; i++) {
+        MaEditorWgt* wgt = ui->getUI(i);
+        if (wgt == nullptr) {
```

Comment:
nit: use `CHECK(wgt != nullptr)` of you expect nullptr or `SAFE_POINT` if you do not.
In this case it looks like it must be SAFEPOINT

Reply:
fixed